### DISCUSSION TITLE
Some issues based on Case 9

### DISCUSSION THREAD

**Discussion by zkzk916** [GeoChemFoam/GeoChemFoam#36] [Q&A] [answered]
Dear Professor Hannah,

Thank you and your team for sharing the code with us to address multiphase flow issues more effectively.

I intend to use the interTransportFoam solver to model a scenario similar to Test Case 9. The key difference is that species T is carried within the wetting phase. Assuming no mass transfer between the phases, I am solely addressing the diffusion of species T in the wetting phase. Therefore, my question is whether the value of the Henry coefficient can be set to 0, but it seems that the case cannot run.

Best regards,
Kai Zhang

**Comment by julienmaes** [comment, marked as answer]
Dear Kai,

H cannot be zero, the best way is to set it to a small value, something like 1e-9 should do it
